packaging/tests/tas-restapi/src/test/java/org/alfresco/rest/rules/UpdateRulesTests.java

```diff
@@ -295,6 +301,107 @@ public void updateRuleFields()
                 .assertThat().field(ID).isNotNull();
     }
 
+    /** Check we can use the POST response and update rule by adding conditions. */
+    @Test (groups = { TestGroup.REST_API, TestGroup.RULES, TestGroup.SANITY })
+    public void updateRuleAddConditions()
+    {
+        final RestRuleModel rule = createAndSaveRule(createRuleModelWithModifiedValues());
+
+        STEP("Try to update the rule and add conditions.");
+        rule.setConditions(createVariousConditions());
+
+        final RestRuleModel updatedRule = restClient.authenticateUser(user).withCoreAPI().usingNode(ruleFolder).usingDefaultRuleSet()
+                .include(IS_SHARED)

```

Comment:
Is the "shared" flag relevant? Maybe it will be better to skip including it in this tests as it's not being asserted in any way. It will make the tests a bit more clear.

Reply:
You're right - I just removed `shared` usage from all added tests.